CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New Feature creation tutorial
 - New Model creation tutorial
 - Added update functionality to the CSV source
+- Git feature to count lines of code
```

Comment:
The description of your PR is what should be added (although this is more of a change than an addition), aka

```
### Changed
- Git feature cloc logs if no binaries are in path
```

Reply:
Thanks! I've made the changes in the next commit. Please check.